docs/UserGuide.md

```diff
@@ -63,27 +74,171 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
 
-   * `clear` : Deletes all contacts.
+1. **Tab Area**: The Tab Area contains File and Help, which allows you to navigate its respective functions.
+2. **CLI**: The Command Line Interface (CLI) is where you can input commands to interact with CareerSync.
+3. **Message Box**: The Message Box displays messages to provide feedback to you.
+4. **Internship List Display**: The Internship List Display displays the details of all your internship applications.
 
-   * `exit` : Exits the app.
+### Fields Summary
+The names of the fields for each internship application is as stated below. When you edit these fields, you will see the changes reflected in the display.
+For the restrictions on what is accepted for each field, kindly refer to [Fields Summary](#fields-summary).
 
-7. Refer to the [Features](#features) below for more details, and have fun using <span style="color:
-#f66a0a;">CareerSync!</span>
+![Fields Annotation](images/introduction/FieldsAnnotation.png)
 
+1. `INDEX` - The index of the internship application in the filtered list.
+2. `COMPANY_NAME` - The name of the company you are applying to.
+3. `ROLE` - The role you are applying for.
+4. `STATUS` - The status of your application.
+5. `DESCRIPTION` - A brief description of the internship.
+6. `LOCATION` - The location of the internship.
+7. `CONTACT_NAME` - The name of the contact person.
+8. `CONTACT_EMAIL` - The email of the contact person.
+9. `CONTACT_NUMBER` - The phone number of the contact person.
 --------------------------------------------------------------------------------------------------------------------
+## Tutorial
+This section would guide you through the basic commands of CareerSync, and how to use them.
+
+
+### Clearing sample internship entries: `clear`
+
+To begin using CareerSync, you should clear the sample internship entries that are present when you first start the 
+application.
+
+Key in the command `clear` in the CLI and press `Enter` to clear the sample internship entries.
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+1. Input:<br>
+   `clear`<br>
+   ![Before clearing](./images/clear/clear-before.png)<br><br>
+
+2. Successful Output after clearing entries:<br>
+   ![After successfully clearing](./images/clear/clear-after.png)
+   There are no more internships in the list.
+</div>
+
+### Adding an internship entry: `add`
+
+Next, let us try to add an internship entry. 
+
+<div markdown="span" class="alert alert-success">
+
+💡 **Try It Out:**<br>
+
+Key in the following command in the CLI: `add /com Tiktok /desc create new recommendation engine /status ongoing /poc jane yeo /email hr@tiktok.com
+/phone 90890301 /loc remote /role Software Intern`
+
+Successful output after adding the entry:<br>
+   ![After successfully adding](./images/tutorial/add.png)
+   You can now see your new internship with the details you entered in the **Internship List Display!**
+</div>
+
```

Comment:
I have this exact thing in my command's section. Unless Im mistaken, I thought I needed to have a try it out within my command section? Should I change the example or have all the try it out within the tutorial?